docs/csharp/programming-guide/classes-and-structs/access-modifiers.md

```diff
@@ -81,7 +83,9 @@ All types and type members have an accessibility level, which controls whether t
  [private](../../../csharp/language-reference/keywords/private.md)   
  [public](../../../csharp/language-reference/keywords/public.md)   
  [internal](../../../csharp/language-reference/keywords/internal.md)   
- [protected](../../../csharp/language-reference/keywords/protected.md)   
+ [protected](../../../csharp/language-reference/keywords/protected.md)
+ [protected internal](../../../csharp/language-reference/keywords/protected-internal.md)
```

Comment:
We need extra white space at the end of these two lines each keyword appears on a different line (look at GitHub preview for this file to see the formatting error.)

Reply:
Thanks for the remark, I pushed a fix for this problem.